src/App.tsx

```diff
@@ -1,61 +1,147 @@
 import './App.scss';
+import { Button, Paper, FormControl } from '@mui/material';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoWithUser } from './types/TodoWithUser';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isUserNotSelected, setIsUserNotSelected] = useState(false);
+
+  const changeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsTitleEmpty(false);
+  };
+
+  const changeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setIsUserNotSelected(false);
+  };
+
+  const todosWithUsers: TodoWithUser[] = todos
```

Comment:
Move this function outside the component, and set the result as the default value in the state